test/Microsoft.AspNetCore.SignalR.Client.Tests/HubConnectionTests.ConnectionLifecycle.cs

```diff
@@ -509,7 +509,10 @@ Task<ConnectionContext> ConnectionFactory(TransferFormat format)
                 {
                     var startTask = hubConnection.StartAsync(cts.Token);
                     var exception = await Assert.ThrowsAnyAsync<OperationCanceledException>(() => startTask.OrTimeout());
-                    Assert.Equal("The operation was canceled.", exception.Message);
+
+                    // REVIEW: Alternatively, we could just not assert the message...
```

Comment:
Thoughts on this? If we don't care about the exact message, we should just not assert it. If we care that it's the "Default" message, we should use the constructor to get the actual string so it's localized.

Reply:
Yeah, I think in this case the exact message is not important. This seems like a fine fix to me

Reply:
I think we don't need this message